packages/blade/docs/tokens/Shadows.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, DocsContainer } from '@storybook/addon-docs';
+import { useTheme, BladeProvider } from '../../src/components';
+import { paymentTheme, bankingTheme } from '../../src/tokens';
+
+<Meta
+  title="Tokens/Shadows"
+  parameters={{
+    docs: {
+      container: ({ children, context }) => {
+        const getThemeTokens = () => {
+          if (context.globals.themeTokenName === 'paymentTheme') {
+            return paymentTheme;
+          }
+          if (context.globals.themeTokenName === 'bankingTheme') {
+            return bankingTheme;
+          }
+          return null;
+        };
+        return (
+          <DocsContainer context={context}>
+            <BladeProvider
+              key={`${context.globals.themeTokenName}-${context.globals.colorScheme}`}
+              themeTokens={getThemeTokens()}
+              colorScheme={context.globals.colorScheme}
+            >
+              {children}
+            </BladeProvider>
+          </DocsContainer>
+        );
+      },
+    },
+  }}
+/>
+
+# 🪞 Shadows
+
+export const Border = () => {
+  const { theme } = useTheme();
+  return (
+    <table>
+      <tbody>
+        {Object.entries(theme.shadows).map(([key, value]) => (
+          <tr key={key} style={{ backgroundColor: '#fafafa' }}>
```

Comment:
Let's put shadows on our actual surface colors instead of this fafafafa. 

```suggestion
          <tr key={key} style={{ backgroundColor: theme.colors.surface.background.level2.lowContrast }}>
```